tests/integration/cmor/_fixes/cordex/test_cordex_fixes.py

```diff
@@ -0,0 +1,204 @@
+                                           long_name='latitude')
+    correct_lon_coord = iris.coords.DimCoord([0.0],
+                                             var_name='lon',
+                                             standard_name='longitude',
+                                             long_name='longitude')
+    wrong_lon_coord = iris.coords.DimCoord([0.0],
+                                           var_name='longitudeCoord',
+                                           standard_name='longitude',
+                                           long_name='longitude')
+    correct_cube = iris.cube.Cube(
+        [[[10.0], [10.0]]],
+        var_name='tas',
+        dim_coords_and_dims=[
+            (correct_time_coord, 0),
+            (correct_lat_coord, 1),
+            (correct_lon_coord, 2)],
+    )
+    wrong_cube = iris.cube.Cube(
+        [[[10.0], [10.0]]],
+        var_name='tas',
+        dim_coords_and_dims=[
+            (wrong_time_coord, 0),
+            (wrong_lat_coord, 1),
+            (wrong_lon_coord, 2)],
+    )
+    return iris.cube.CubeList([correct_cube, wrong_cube])
+
+
+@pytest.fixture
+def cordex_cubes():
+    coord_system = iris.coord_systems.RotatedGeogCS(
+                grid_north_pole_latitude=39.25,
+                grid_north_pole_longitude=-162
+    )
+    time = iris.coords.DimCoord(np.arange(0, 3),
+                                var_name='time',
+                                standard_name='time')
+
+    rlat = iris.coords.DimCoord(np.arange(0, 412),
+                                var_name='rlat',
+                                standard_name='grid_latitude',
+                                coord_system=coord_system,
+                                )
+    rlon = iris.coords.DimCoord(np.arange(0, 424),
+                                var_name='rlon',
+                                standard_name='grid_longitude',
+                                coord_system=coord_system,
+                                )
+    lat = iris.coords.AuxCoord(np.ones((412, 424)),
+                               var_name='lat',
+                               standard_name='latitude')
+    lon = iris.coords.AuxCoord(np.ones((412, 424)),
+                               var_name='lon',
+                               standard_name='longitude')
+
+    cube = iris.cube.Cube(
+        np.ones((3, 412, 424)),
+        var_name='tas',
+        dim_coords_and_dims=[
+            (time, 0),
+            (rlat, 1),
+            (rlon, 2)],
+        aux_coords_and_dims=[
+            (lat, (1, 2)),
+            (lon, (1, 2))
+        ]
+
+    )
+    return iris.cube.CubeList([cube])
+
+
+def test_mohchadrem3ga705_fix_metadata(cubes):
+    fix = MOHCHadREM3GA705(None)
+    out_cubes = fix.fix_metadata(cubes)
+    assert cubes is out_cubes
+    for cube in out_cubes:
+        for coord in cube.coords():
+            right_coord = cubes[0].coord(coord)
+            assert coord == right_coord
```

Comment:
Yes but cubes[0] is always the correct one, so I am comparing the output of the check to a cube that has not been changed because it's correct from the beginning.

Reply:
It would be good to add a comment to the code to explain that. Like that, the information is available for future developers.

Reply:
I see, I have updated the test then. It's parametrising now